topics/ui/controls/link.md

```diff
@@ -10,127 +10,266 @@
 
 </tldr>
 
-![](01_link_example.png){width=120}
+![A preview of link types: default, drop-down, and external.](link.png){width=706}
 
 ## When to use
 
-Use a regular link for navigation between pages of the same window.
-
-![](02_deployment_server.png){width=494}
+### Navigation between pages of the same window
 
-*This option is in the Settings dialog. "Configure servers" opens another page of the same dialog.*
-
-![](03_manage_scopes.png){width=411}
+Use a regular link for navigation between pages of the same window.
 
-*Do not use a button to navigate to another page of the same window.*
+<table style="none" border="false">
+  <tr>
+    <td>
+        <format color="369650" style="bold">Correct</format>
+        <img src="link_when_to_1_correct.png" width="378" alt="A vertical list of three options: 'Non-Project Files', 'Problems', and 'Production', followed by a 'Manage scopes…' link below."/>
+    </td>
+    <td>
+        <format color="E55765" style="bold">Incorrect</format>
+        <img src="link_when_to_1_incorrect.png" width="378" alt="A vertical list of three options: 'Non-Project Files', 'Problems', and 'Production', followed 'Manage scopes' button below."/>
+    </td>
+  </tr>
+</table>
+
+### Web resources
 
 Use an external link for navigation to web resources. See below for the [external link icon](#external-link-icon).
 
-![](04_external_link.png){width=248}
+![Text 'Learn more about what is logged' with 'what is logged' as an external link with an arrow icon.](link_when_to_2.png){width=706}
 
-Use a regular or [drop-down link](#drop-down-link) for secondary actions in packed or small UI areas.
-
-![](05_action_regular_link.png){width=367}
+### Secondary actions in packed UI areas
 
-*The "Reset" action is a link for two reasons: (1) The action appears only when the default value in the field is changed and reverting to a default is considered a rare scenario. (2) A lightweight link fits better than a button in this busy layout.*
+Use a regular or [drop-down link](#drop-down-link) for secondary actions in packed or small UI areas.
 
-![](06_action_dropdown.png){width=550}
+For example, the <control>Reset</control> action bellow is a link for two reasons:
+* The action appears only when the default value in the field is changed, and reverting to a default is considered a rare scenario.
+* A lightweight link fits better than a button in this busy layout.
 
-*The "Modify options" drop-down link fits into the top right corner of a busy layout.*
+![Form with three settings: 'Hard wrap at' with a Reset link, 'Wrap on typing', and 'Visual guide'.](link_when_to_3.png){width=706}
```

Comment:
The illustration doesn't look quite busy. Possibly just leave the example with the "Modify options" link?